src/bin/tikv-ctl.rs

```diff
@@ -911,6 +929,24 @@ fn main() {
                         .help("PD endpoints"),
                 ),
         )
+        .subcommand(
+            SubCommand::with_name("unsafe-conf-change")
+                .about("force conf-change on a store, for majority of peers failed")
+                .subcommand(
+                    SubCommand::with_name("remove-stores")
```

Comment:
Is there any other sub-command?

Reply:
For now, not more other sub-commands.

Reply:
Then the command can be removed. Or you can rename the `unsafe-conf-change` to `unsafe-recover`.